.travis.yml

```
@@ -14,6 +14,9 @@ branches:
 _integration_test: &integration_test
   os: linux
   language: minimal
+  before_install:
+    #Skip integration tests if changes are only in the `docs` directory. This only works for a pull-request based development workflow like Skaffold's. 
```

Nit: insert space after the comment.  WDYT of this shorter alternative:
```suggestion
    # Skip integration tests for `docs`-only changes (only works for PR-based dev workflows like Skaffold's).
```